scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -578,16 +578,10 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
 
             /**
              * Outdent is needed in following cases:
-             *  - If indentation on next line is less than current
-             *    and previous token can't continue expr on the next line
-             *  - At the end of `match` block even if indentation level is not changed
-             *    Example:
-             *    ```
```

Comment:
the ` ``` ` fence can't be more than 3 spaces in, or it's not a fence. remove indent or replace with `{{{` which always works.

Reply:
It might be extended in case of list item - https://spec.commonmark.org/0.29/#example-233 so that a parser issue

Reply:
@tgodzik @kitbellew I've noticed that [`Scaladoc.ListItem`](https://github.com/scalameta/scalameta/blob/main/scalameta/trees/shared/src/main/scala/scala/meta/internal/Scaladoc.scala#L100) doesn't accept anything except `Text`or nested `List`.

Would it be ok to change signature of `ListItem`?

Reply:
sure. do you want to take a paragraph? :)